scripts/populate-popular-and-new-and-noteworthy-projects.py

```diff
@@ -0,0 +1,140 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
```

Comment:
The referenced PR uses the values via `website.settings`. It doesn't import them from `website.settings.default`, even though they're defined there.

Have you had occasion to read/grok [`website/settings/__init__.py`](https://github.com/CenterForOpenScience/osf.io/blob/03034779d421991e46f85a8ef1af3b07306228a8/website/settings/__init__.py) yet? If not, I think that could help clear up what I'm talking about here. :)